src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -52,6 +51,16 @@ class DockerBuildImageFunctionalTest extends AbstractFunctionalTest {
         result.output.contains("label1:test1, label2:test2")
     }
 
+    def "can build image with labels without using build-args"() {
```

Comment:
We should merge the test cases "can build image with build variables" and "can build image with labels without using build-args". Both execute the same task and assert the same output. Let's use a single test method with a `where` clause. Annotate the method with `@Unroll`.